components/SearchSuggestions/SearchSuggestions.scss

```diff
@@ -68,6 +68,10 @@
 		padding: 0;
 		padding-bottom: 5px;
 
+		li.selected {
+			background-color: $gray-light;
```

Comment:
Okies. But managing the separate branch till we have UI kit complete, can be bit difficult because we have continuous updates in dev branch. Anyway, it sounds good to have finished nav out instead of unfinished one.